rtil/src/threads/ue/rebo/rebo_init.rs

```diff
@@ -372,6 +376,21 @@ fn store_settings(settings: Map<String, String>) {
 }
 
 #[derive(rebo::ExternalType, Serialize, Deserialize)]
+struct Recording {
+    version: i32,
+    author: String,
+    steam_id: u64,
+    filename: String,
+    frame_count: i64,
+    recording_start_timestamp: u64,
+    recording_end_timestamp: u64,
+    recording_save_timestamp: u64,
```

Comment:
Thinking about it, I think it makes sense here to use RFC 3339 / ISO8601 including the timezone. The timezone can help display correct times in case we ever develop a recording-upload-website. It might also make sense to convert the u64 gotten from rebo into the same thing. Take a look at [`chrono`](https://docs.rs/chrono/latest/chrono/) for proper time handling.